protocol/src/test/java/org/opensearch/sql/protocol/response/format/CsvResponseFormatterTest.java

```diff
@@ -130,4 +130,9 @@ void replaceNullValues() {
     assertEquals(format(expected), formatter.format(response));
   }
 
+  @Test
+  void testContentType() {
+    assertEquals(formatter.contentType(), "plain/text; charset=UTF-8");
```

Comment:
if you use CONTENT_TYPE here, then your tests will pass even if the header text changes.

Reply:
good idea, implemented in https://github.com/Bit-Quill/opensearch-project-sql/pull/279/commits/1fce3446613b66bee8476b87f8d8a5f5da7bb66e